docs/appendices/release-notes/5.3.5.rst

```diff
@@ -60,3 +60,10 @@ Fixes
     => IndexOutOfBoundsException[Index: 1 Size: 1]
     SELECT * FROM test AS T WHERE T.o2['unknown_col'] IS NOT NULL;
     => IndexOutOfBoundsException[Index: 1 Size: 1]
+
+- Fixed an issue which caused INSERT INTO statements to skip generated
```

Comment:
since you're adjusting this: how about using double backticks around `INSERT INTO`, `NULL` and `CHECK`? or even `ref` to the doc sections? (if you do so, please adjust the other 2 release notes files)

Reply:
Sounds good, I added ticks and added a clarification to the "abandoned partitions" that it's only about INSERT ...SELECT, insert from values doesn't have such issue as it calls validation before creating an index.

I think ticks are enough, we already had similar entry changes with ticks and without refs.

Reply:
thx!